test/CounterImplementation.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import { UpgradeableProxied } from "../src/UpgradeableProxied.sol";
+
+contract CounterImplementation is UpgradeableProxied {
+
+    uint256 public number;
+
+    mapping(bytes32 => uint256) public numberMapping;
+
+    modifier auth {
+        require(wards[msg.sender] == 1, "CounterImplementation/not-authed");
```

Comment:
Should be `not-authorized` instead of `not-authed`. This was a mistake in the cropper repo.

Reply:
Updated